### DISCUSSION TITLE
Ingestion Error

### DISCUSSION THREAD

**Discussion by ankit289** [Sunbird-cQube/community#852] [General]
![image](https://github.com/Sunbird-cQube/community/assets/43002190/d0a26551-914e-483f-9bfa-17c6417ca1ef)

Facing this error while ingesting data in school-infra. Checked cluster,school and school category data and grammar files , all are good

**Comment by dhanush-2397** [comment]
@ankit289 The table name should not contain hyphens "-". Instead of school-infra you can give school_infra.
Please change the namespeace value in config.json.
Example:
 {
      "name": "School Attendance",
      **"namespace": "sch_att",**
      "description": "School Attendance",
      "shouldIngestToDB": true,
      "input": {
        "files": "./ingest/VSK/programs/school-attendance"
      },
      "./output": {
        "location": "././output/programs/school-attendance"
      },
      "dimensions": {
        "whitelisted": [
        ],
        "blacklisted": []
      }
    },
